apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/list-02.php

```diff
@@ -14,7 +14,7 @@
 <!-- /wp:spacer -->
 
 <!-- wp:paragraph {"style":{"typography":{"fontSize":72,"lineHeight":"0.9"}}} -->
-<p style="line-height:.9;font-size:72px;"><p style="line-height:.9;font-size:72px;"><strong>%1$s<br>%2$s<br>%3$s </strong><br><strong>%4$s<br>%5$s<br>%6$s<br>%7$s<br>%8$s<br>%9$s</strong></p></p>
+<p style="line-height:.9;font-size:72px;"></p><p style="line-height:.9;font-size:72px;"><strong>%1$s<br>%2$s<br>%3$s </strong><br><strong>%4$s<br>%5$s<br>%6$s<br>%7$s<br>%8$s<br>%9$s</strong></p><p></p>
```

Comment:
Wouldn't this make two empty paragraphs and make it three paragraphs in total?

The original markup was also weird though, it looks like a paragraph wrapped in a paragraph. And looking at the inline style, the paragraph tag `<p style="line-height:.9;font-size:72px;">` has been duplicated in the markup for some reason?